Orange/widgets/visualize/owscatterplot.py

```diff
@@ -380,10 +380,24 @@ def update_graph(self, attributes=None, reset_view=True, **_):
         self.graph.zoomStack = []
         if attributes and len(attributes) == 2:
             self.attr_x, self.attr_y = attributes
+        # print (self.attr_x)
+        # print (self.attr_y)
+        # print (self.data[0])
+        idx = self.data.domain.index(self.attr_x)
+        idy = self.data.domain.index(self.attr_y)
+        flag = 0
+        for x in self.data[:,idx]:
```

Comment:
where to use the above code which you mentioned above? I mean which line to be removed